src/module.c

```diff
@@ -12244,6 +12244,138 @@ int RM_LoadConfigs(RedisModuleCtx *ctx) {
     return REDISMODULE_OK;
 }
 
+/* --------------------------------------------------------------------------
+ * ## RDB load/save API
+ * -------------------------------------------------------------------------- */
+
+#define REDISMODULE_RDB_STREAM_FILE 1
+
+typedef struct RedisModuleRdbStream {
+    int type;
+
+    union {
+        char *filename;
```

Comment:
why not use sds ?

Reply:
As we didn't use any of sds capabilities for this string in the PR, I just went with `char*`.